ckan/tests/test_coding_standards.py

```diff
@@ -953,4 +955,4 @@ def test_bad(self):
                ' resolving\nWe should not be translating exceptions in most'
                ' situations.  We need to when the exception message is passed'
                ' to the front end for example validation')
-        show_fails(msg, self.fails)
+        show_fails(msg, self.fails)
```

Comment:
Found it - looks like you got rid of the final \n, which is what the PEP8 error is about. I'll commit the fix.